python/ray/serve/controller.py

```diff
@@ -300,32 +283,95 @@ def _scale_backend_replicas(self, backends: Dict[BackendTag, BackendInfo],
 
                 self.backend_replicas_to_stop[backend_tag].append(replica_tag)
 
-    async def _stop_pending_backend_replicas(self) -> None:
-        """Stops the pending backend replicas in self.backend_replicas_to_stop.
+    async def _enqueue_pending_scale_changes_loop(self,
```

Comment:
prefer to separate this into separate methods for create/stop

Reply:
I can go back to this, but I wanted to centralize these to one method because this more closely resembles what will be the 'outer control loop' . I'll update the comment for this PR with what the eventual goal will look like.

Reply:
Sure, but in the "outer control loop" we'd probably want to wrap these in two separate methods for readability anyways. Fine to leave it like this for now.

Reply:
Sure, but in the "outer control loop" we'd probably want to wrap these in two separate methods for readability anyways. Fine to leave it like this for now.